hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/src/main/java/com/hedera/hcsapp/controllers/CreditsController.java

```diff
@@ -173,7 +173,7 @@ public CreditsController() throws Exception {
             
             credit = creditRepository.save(credit);
             new OutboundHCSMessage(appData.getHCSCore())
-                  .overrideEncryptedMessages(false)
+                  .overrideEncryptedMessages(true)

```

Comment:
See other comment wrt override, think we need to raise a separate issue on this.

Reply:
See issue #371